frequency-chain/style-guide
# 10 desktop header  navigation

# Problem
Desktop and Mobile header and navigation.

closes #10  
closes #5 

# Solution

Added mobile and desktop header and navigation components and stories

## Steps to Verify:

1. run storybook to make sure it looks good
2. read through code to make sure it is scalable and has all of the functionality that we could want.

## Screenshots (optional):

https://github.com/user-attachments/assets/2c58a66e-3639-4ce8-a275-c30a9772c704
